draft-irtf-cfrg-opaque.md

```diff
@@ -830,20 +763,31 @@ Steps:
 2. N = Unblind(input.data_blind, Z)
 3. y = Finalize(PwdU, N, "RFCXXXX")
 4. RwdU = Harden(y, params)
-5. credentials_key = HKDF(salt=0, IKM=RwdU, "CredentialsKey", Nk)
-6. exporter_key = HKDF(salt=0, IKM=RwdU, "ExporterKey", Nk)
-7. n = random(Nn) # generate a fresh nonce randomly
-8. Credentials C with (skU=skU, pkS=response.pkS)
-9. (ct, t) = Seal(credentials_key, n, "", C)
-10. EnvU = concat(n, ct, t)
-11. Create RegistrationUpload upload with envelope value EnvU
-11. Output (upload, exporter_key)
+5. Create Credentials C according to server policy
+6. pt = SerializeExtensions(C.secret_credentials)
+7. nonce = random(Nn)
+8. pseudorandom_pad = HKDF-Expand(RwdU, contact(nonce, "Pad"), len(pt))
```

Comment:
Make reference to RFC 5869 about the roles of the three inputs to HKDF-Expand, namely, key, info, and output length.

Reply:
I added a reference at the end of this algorithm.